### PR TITLE
docs(examples): usage of react-router, resolves #795

### LINKED ISSUE #795: [Docs] Create documentation examples for React Router with common navigational components
Would be great if we could provide documentation around, or examples of, usage of common navigational components with React Router. For example:

- Breadcrumb
- Tabs
- ILN

### PR DESCRIPTION
**Pull request by alisonemiller**
Closes IBM/carbon-components-react#795

Added an example app of using `react-router` with the `Breadcrumb` and `Tab` components.

#### Changelog

**New**

- added `react-router` app under the `examples` directory (bootstrapped with `create-react-app`)

### CODE REVIEW

**Comment by alisonemiller** [comment]
Hey, there. I see the `yarn ci-check` failed on my PR. Can someone point me in the right direction on what I need to do there to resolve it? Thanks!

**Comment by asudoh** [comment]
Hi 👋 thank you for the contribution! I haven't been able to run https://codesandbox.io/s/github/alisonemiller/carbon-components-react/tree/doc-react-router-example/examples/react-router somehow, but I hope some reviewers can and see it looks good.

Wrt CI error, the cause is:

```
$ prettier --list-different "**/*.{scss,css,js,md}"
examples/react-router/src/App.js
examples/react-router/src/App.scss
examples/react-router/src/index.css
examples/react-router/src/index.js
```

Could you try running `yarn prettier:staged` against those files? Thanks!

**Comment by emyarod** [comment]
unfortunately I also was not able to load the Code Sandbox example. In the original issue thread I created a possible [demo](https://github.com/IBM/carbon-components-react/issues/795#issuecomment-395569605) as well. It might be useful to reference that, but I didn't receive any feedback after posting it

**Comment by alisonemiller** [comment]
Thanks for the feedback. I was running the sample against my local machine; I'll have to look into why it's not working on code sandbox (probably won't get around to it until later this week).

**Comment by alisonemiller** [comment]
@emyarod and @asudoh - I fixed the `yarn ci-check` failure. The Code Sandbox also appears to be working now (although I did not make any changes related to that): https://codesandbox.io/s/github/alisonemiller/carbon-components-react/tree/doc-react-router-example/examples/react-router.

Please take another look when you get a chance - thanks!

**Comment by asudoh** [comment]
@emyarod Does @alisonemiller's latest code make sense to you? Any suggestions, esp from your earlier work? Thanks!

File: examples/react-router/src/logo.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
[the remark this answers is not in the document]
Comment (alisonemiller) [reply examples/react-router/src/logo.svg]: @emyarod, agreed. i removed it.

**Comment by asudoh** [comment]
@emyarod Does @alisonemiller's latest code look good for you...? Thanks!

**Comment by emyarod** [comment]
makes sense but I cannot get the Code Sandbox to properly display the demo. I think if the issues around that persist we can open a new ticket for that later on

**Comment by asudoh** [comment]
@alisonemiller It appears [your custom Sass build for `carbon-components`](https://github.com/IBM/carbon-components-react/blob/a09c86c9e02c2a77ffeabe4ab1c2f1a12a787307/examples/react-router/src/App.js#L5) is causing huge number of downloads, looking like taking forever to get the application running. Can we [use a pre-built CSS like our base example](https://github.com/IBM/carbon-components-react/blob/v6.60.0/examples/codesandbox/public/index.html#L12)? Thanks!

**Comment by alisonemiller** [comment]
@asudoh good call. Fixed!

**Pull request merged**